data_checks/checks/epre/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+
+from .budget_phase_required_values import BudgetPhaseRequiredValues
+
+
+checks = ['budget-phase-required-values']
```

Comment:
Pi/JD, I'm a bit uncertain about this. Not sure where to store a list with the available checks for a dataset type. Do you think this is okay?

Reply:
ja this looks like a good place at the moment